src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -17328,6 +17328,12 @@ std::string ConvertToEscaped(std::string const &inString, bool isXML) // Input S
             s += "&gt;";
         } else if (c == char(176)) {
             s += "&deg;";
+        } else if (c == char(226) && char(inString[index]) == char(137) && char(inString[index + 1]) == char(164)) { // ≤
+            s += "&le;";
+            index += 2;
+        } else if (c == char(226) && char(inString[index]) == char(137) && char(inString[index + 1]) == char(165)) { // ≥
+            s += "&ge;";
+            index += 2;
```

Comment:
Thanks for changing this; I realize it's technically the same thing, but this is one less thing to grok when looking at this code.